packages/app-desktop/gui/NoteEditor/NoteBody/CodeMirror/utils/index.ts

```diff
@@ -70,8 +70,13 @@ export function useScrollHandler(editorRef: any, webviewRef: any, onScroll: Func
 
 		if (editorRef.current) {
 			const percent = editorRef.current.getScrollPercent();
-
-			setViewerPercentScroll(percent);
+			if (percent) {
```

Comment:
With this change, doesn't it mean that if you want to scroll to the top (percent = 0), you cannot anymore? Maybe we should check using `!isNaN(percent)` instead?

Reply:
Good point, thanks. Fixed